3-semantic-html/basic.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav>
+<ul>
+    <li>
+        <a href="">Home</a>
+    </li>
+    <li>
+        <a href="#artistas">Artistas</a>
+    </li>
+    <li>      
+        <a href="#digital">Servicios digitales</a>
+    </li>
+    <li>
+        <a href="">Eventos</a>
+    </li>
+</ul>
+        </nav>
+    </header>
```

Comment:
You can **indent** your tags, in that way it would be easy to read and understand.
```suggestion
<header>
        <nav>
            <ul>
                <li>
                    <a href="">Home</a>
                </li>
                <li>
                    <a href="#artistas">Artistas</a>
                </li>
                <li>
                    <a href="#digital">Servicios digitales</a>
                </li>
                <li>
                    <a href="">Eventos</a>
                </li>
            </ul>
        </nav>
    </header>
```